client/src/components/Workflow/Run/WorkflowRunFormSimple.vue

```diff
@@ -37,9 +67,12 @@ export default {
         },
     },
     data() {
+        const newHistory = this.targetHistory == "new" || this.targetHistory == "prefer_new";
         return {
             formData: {},
             inputTypes: {},
+            sendToNewHistory: newHistory,
+            useCachedJobs: this.useJobCache, // TODO:
```

Comment:
Is a to-do missing here? Or can this be removed?

Reply:
Hmm, I don't know what would be the right thing here in combination with v-model. The way you compute newHistory within `data()` looks odd to me and I wouldn't do it if there was no v-model, which is why I thought this isn't quite right. I guess you wouldn't want newHistory to be reactive, so maybe this is the right thing to do ?